src/lib.rs

```diff
@@ -163,26 +166,23 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone, V: Clone {
         })
     }
 
-    /// Check for existence of a key
-    // FIXME: Deprecated in favor of the `contains_key` function defined below.
```

Comment:
As this is an API change we will need to publish a release version change when this is merged.